File: src/main/kotlin/com/example/demo/MovieInfo.java
package com.example.demo;

public class MovieInfo {

    private String characterName;
    private String movieName;
    private String year;
    private String url;

    public String getCharacterName() {
        return characterName;
    }

    public void setCharacterName(String characterName) {
        this.characterName = characterName;
    }

    public String getMovieName() {
        return movieName;
    }

    public void setMovieName(String movieName) {
        this.movieName = movieName;
    }

    public String getYear() {
        return year;
    }

    public void setYear(String year) {
        this.year = year;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    @Override
    public String toString() {
        final StringBuilder sb = new StringBuilder("MovieInfo{");
        sb.append("characterName='").append(characterName).append('\'');
        sb.append(", movieName='").append(movieName).append('\'');
        sb.append(", year='").append(year).append('\'');
        sb.append(", url='").append(url).append('\'');
        sb.append('}');
        return sb.toString();
    }
}

File: src/main/kotlin/com/kapresoft/demo/MovieInfo.kt
package com.kapresoft.demo

class MovieInfo {

    var characterName: String? = null
    var movieName: String? = null
    var year: String? = null
    var url: String? = null

    override fun toString(): String {
        val sb = StringBuilder("MovieInfo{")
        sb.append("characterName='").append(characterName).append('\'')
        sb.append(", movieName='").append(movieName).append('\'')
        sb.append(", year='").append(year).append('\'')
        sb.append(", url='").append(url).append('\'')
        sb.append('}')
        return sb.toString()
    }

}